packages/ComponentLibrary/src/components/Modal/modal.styles.ts

```diff
@@ -0,0 +1,12 @@
+const modalStyles = {
```

Comment:
The names of the **colors** and **measurements** must be imported from a _**Constants Utility**_